.travis.yml

```diff
@@ -1,9 +1,14 @@
+---
```

Comment:
unrelated change

```suggestion
```

Reply:
Its yaml headers recommended by linters should i remove it ?

Reply:
Removed in https://github.com/ruby/www.ruby-lang.org/pull/1420/commits/fea533eb88543172cf6acc8842b2efebd7f96db0